pkg/operator/operator.go

```diff
@@ -121,23 +124,23 @@ func NewOperator() (context.Context, *Operator) {
 	})
 
 	// Client Config
-	config := controllerruntime.GetConfigOrDie()
+	config := ctrl.GetConfigOrDie()
 	config.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(float32(options.FromContext(ctx).KubeClientQPS), options.FromContext(ctx).KubeClientBurst)
 	config.UserAgent = fmt.Sprintf("%s/%s", appName, Version)
 
 	// Client
 	kubernetesInterface := kubernetes.NewForConfigOrDie(config)
 
 	// Logging
-	logger := logging.NewLogger(ctx, component)
-	ctx = knativelogging.WithLogger(ctx, logger)
+	logger := log.FromContext(ctx).WithName(component)
+	ctx = logr.NewContext(ctx, logger)
 	logging.ConfigureGlobalLoggers(ctx)
 
-	knativelogging.FromContext(ctx).With("version", Version).Debugf("discovered karpenter version")
+	log.FromContext(ctx).WithValues("version", Version).Info("discovered karpenter version")
```

Comment:
Note: This is a change in behavior from the Debug log, but I'm actually fine with this change in behavior. Seems reasonable to me